corehq/apps/mobile_auth/models.py

```diff
@@ -44,3 +47,43 @@ def key_for_time(cls, domain, user_id, now):
             include_docs=True,
         ).first()
         return key_record
+
+    @classmethod
+    def _migration_get_sql_model_class(cls):
+        return SQLMobileAuthKeyRecord
+
+    @classmethod
+    def _migration_get_fields(cls):
+        return ["domain", "user_id", "valid", "expires", "type", "key"]
+
+
+def _default_uuid():
+    return uuid.uuid4().hex
+
+
+def _default_key():
+    return generate_aes_key().decode('utf-8')
+
+
+class SQLMobileAuthKeyRecord(SyncSQLToCouchMixin, models.Model):
+    uuid = models.UUIDField(primary_key=True, db_index=True, default=_default_uuid)
```

Comment:
Can we revisit the decision not to name primary key fields `id`? I would vastly prefer that we always the name `id` for the primary key field unless the model has a composite primary key. I think it's a good convention to follow, and the cost of not following it is that it will make it harder to intuit the name of the primary key for models with inconsistent naming.

Reply:
Since we had that conversation recently, I'd prefer it not block this PR - unless it would be difficult in the future to rename `uuid` to `id`.

Reply:
Offlined about this, I'm going to update to use `db_column="id"` so that it'll be easy in the future to rename the column but so I don't have to update `uuid` references in the code right now.